fiona/__init__.py

```diff
@@ -202,6 +202,16 @@ def open(fp, mode='r', driver=None, schema=None, crs=None, encoding=None,
     crs_wkt : str
         An optional WKT representation of a coordinate reference
         system.
+    ignore_fields : list
+        List of field names to ignore on load.
+    ignore_geometry : bool
+        Ignore the geometry on load.
+    include_fields : list
```

Comment:
These appear to be unrelated to this MR, but were missing from the docstring.

Reply:
Figured so!